### PR TITLE
Added csstype for better typing experience

### PR DESCRIPTION
**Pull request by samuelcastro**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -100,6 +100,7 @@
     "cookieconsent": "3.1.1",
     "cookies": "0.8.0",
     "css-to-react-native": "3.0.0",
+    "csstype": "^3.0.3",
```
Comment (Vadorequest) [inline package.json]: Please use fixed versions.

I recommend running `npm config set save-exact=true`, see https://stackoverflow.com/questions/40105268/how-to-force-npm-to-install-specific-version-of-package-by-default

File: src/components/assets/GraphCMSAsset.test.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ describe('GraphCMSAsset', () => {
         const id = 'test';
         const title = 'Test asset';
         const classes = 'test-class1 test-class2';
-        const style = { paddingTop: 5 };
+        const style = { paddingTop: '5' };
```
Comment (Vadorequest) [inline src/components/assets/GraphCMSAsset.test.tsx]: A number should be a valid input, I'd prefer not to change those kinds of things if it's avoidable (reduce work between presets)

Comment (Vadorequest) [reply src/components/assets/GraphCMSAsset.test.tsx]: I think it can be avoided, see https://github.com/frenic/csstype#generics

**Review by Vadorequest — CHANGES_REQUESTED** [review]

**Comment by Vadorequest** [comment]
@samuelcastro Thanks! I added a few comments above :)

File: src/components/assets/GraphCMSAsset.test.tsx (hunk 2 above)
Comment (samuelcastro) [reply src/components/assets/GraphCMSAsset.test.tsx]: Yeah it can be avoided, I just wanted to use the standard pattern, I'll change it.

**Comment by Vadorequest** [comment]
Thanks!

**Review by Vadorequest — APPROVED** [review]

**Comment by Vadorequest** [comment]
@samuelcastro Funnily enough, I encounter more issues with that new typing.

```
TS2322: Type '{ alignContent: string; alignItems: string; alignSelf: string; animationDelay: string; animationDirection: string; animationDuration: string; animationFillMode: string; animationIterationCount: string | number; ... 750 more ...; vectorEffect: VectorEffectProperty; }' is not assignable to type 'CSSProperties'.   Types of property 'appearance' are incompatible.     Type 'AppearanceProperty' is not assignable to type 'Appearance'.       Type '"button-bevel"' is not assignable to type 'Appearance'.
```

Etc. Ran into half a dozen of issues, while the former code was working fine.

**Comment by samuelcastro** [comment]
Nooo, how to reproduce?

**Comment by Vadorequest** [comment]
I went back to the current code and the error doesn't show anymore. 🤷 

Maybe a TS sync issue, I really don't get it. If I reproduce it I'll let you know.

**Pull request merged**